apertium/stream_5_3_3_tagger_trainer.h

```diff
@@ -26,7 +26,7 @@ namespace Apertium {
 class Stream_5_3_3_TaggerTrainer : private basic_5_3_3_Tagger,
                                    public basic_StreamTaggerTrainer {
 public:
-  Stream_5_3_3_TaggerTrainer(const Flags &Flags_);
+  Stream_5_3_3_TaggerTrainer(Flags &Flags_);
```

Comment:
This and others don't seem necessary?

Reply:
I started with `basic_tagger.h`, and made changes according to the errors reported